package.json

```diff
@@ -5,7 +5,8 @@
   "version": "2.0.0-alpha",
   "description": "Take your passwords everywhere.",
   "engines": {
-    "firefox": ">= 63"
+    "firefox": ">= 63",
+    "node" : ">=11.6.0"
```

Comment:
Is 11.6.0 the *minimum* version that works here? I've got 10.12.0 installed, and I think that's been working for me.

I'd kind of rather stick closer to the current LTS - which looks like 10.15.0 right now.